contract_execution_result.js

```diff
@@ -0,0 +1,51 @@
+const {AssetProof} = require('./asset_proof.js');
+
+/**
+ * To handle ContractExecutionResponse
+ */
+class ContractExecutionResult {
+  /**
+   * @param {Object} result
+   * @param {Array} proofs
+   */
+  constructor(result, proofs) {
+    this.result = result;
+    this.proofs = proofs;
+  }
+
+  /**
+   * @param {gRPCContractExecutionResponse} response
```

Comment:
There is no `gRPCContractExecutionResponse` so should it simply be `ContractExecutionResponse` or `proto.rpc. ContractExecutionResponse` ?

Reply:
`gRPCContractExecutionResponse` is probably not a good name.
It doesn't really denote which class it refers to.
It seems to me that `proto.rpc.ContractExecutionResponse` is a better naming.